lib/bee/writer/Neo4jWriter.rb

```diff
@@ -2,10 +2,18 @@
 
 module Bee
   class Neo4jWriter < Writer
-    def initialize(db_path=".beedb")
-      @session = Neo4j::Session.open(:embedded_db, db_path)
-      @session.start
+    #if db_path is empty, we take server, otherwise embedded
+    def initialize(db_path=".beedb",label="")
+      if (db_path.empty?)
+        @session = Neo4j::Session.open(:server_db)
+        @embedded = false
+      else
+        @session = Neo4j::Session.open(:embedded_db, db_path)
+        @session.start
```

Comment:
No, it seems like start() does not work for non-embedded DBs (method not implemented).